.github/workflows/docker-tags.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Docker image for tag
+
+on:
+  push:
+    tags:
+      # Since this action pushes just a tagged image it is only active on tags. We exclude `-dev` tags.
+      - 'v*'
+      - '!v*-dev'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Get Version
+        id: version
+        run: echo "::set-output name=RELEASE_VERSION::$(cat VERSION)"
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile
+          push: true
+          tags: |
+            zeekurity/zeek:${{ steps.version.outputs.RELEASE_VERSION }}
```

Comment:
We could add one quick step here validating that e.g., executables in the image work (e.g., all dynamic library or Python module deps present).

Here and for other workflows as well.

Reply:
I added a number of basic container structure tests which we run before publishing images, and also a dedicated GH workflow to validate that the image is not broken in PRs.